### PR TITLE
SEARCH 593: Update MB Solr tests to work with 3.1

### PR DESCRIPTION
**Pull request by amCap1712**
1) Increase timeout to 15s
2) Change cdstub comment field to disambiguation
3) Remove ended field from Place resources
4) Remove packaging field from Release resources
5) Remove name field from Recording doc

### CODE REVIEW

File: mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml [hunk 1]
```diff
@@ -1 +1 @@
...
.../name><sort-name>Chiptunes = WIN \m|♥|m/</sort-name><alias-list><alias sort-name="Chiptunes = WIN">Chiptunes = WIN</alias></alias-list></artist></name-credit></artist-credit><release-group id="268d91a0-1a7f-4600-bfd4-74d738eb7de4" type="Compilation"><primary-type>Album</primary-type><secondary-type-list><secondary-type>Compilation</secondary-type></secondary-type-list></release-group><date>2012-06-05</date><country>XW</country><release-event-list><release-event><date>2012-06-05</date><area id="525d4e18-3d00-31b9-a58b-a146a916de8f"><name>[Worldwide]</name><sort-name>[Worldwide]</sort-name><iso-3166-1-code-list><iso-3166-1-code>XW</iso-3166-1-code></iso-3166-1-code-list></area></release-event></release-event-list><barcode/><label-info-list><label-info><label id="7669cf9a-7328-44cf-96ba-b2dc9d534ed8"><name>Chiptunes = WIN \m|♥|m/</name></label></label-info></label-info-list><medium-list count="1"><track-count>51</track-count><medium><format>Digital Media</format><disc-list count="0"/><track-list count="51"/></medium></medium-list><tag-list><tag count="1"><name>bitunes</name></tag><tag count="1"><name>bandcamp</name></tag><tag count="2"><name>chiptunes</name></tag></tag-list></release>
```
Comment (amCap1712) [inline mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml:1]: There seems to a regression with the `packaging` field.
*Production Behavior*
```json
packaging: "None",
packaging-id: "119eba76-b343-3e02-a292-f0f00644bb9b",
``` 

*Current Behavior*
```json
packaging: {
 "name": "None",
  "id": "119eba76-b343-3e02-a292-f0f00644bb9b"
}
```

File: mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/place-list.xml [hunk 2]
```diff
@@ -1 +1 @@
...
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?><metadata xmlns="http://musicbrainz.org/ns/mmd-2.0#" xmlns:ns2="http://musicbrainz.org/ns/ext#-2.0"><place-list count="1" offset="0"><place id="fd60135e-d173-4a56-87fd-1f1c4bc62bd1" type="Venue" ns2:score="100"><name>Ilmbühne</name><coordinates><latitude>50.99508</latitude><longitude>11.36062</longitude></coordinates><area id="c7644e45-dec4-43fd-aad6-35036b8e911d"><name>Weimar</name><sort-name>Weimar</sort-name></area><alias-list><alias sort-name="Lindenrondell" type="Search hint">Lindenrondell</alias></alias-list></place></place-list></metadata>
```
Comment (amCap1712) [inline mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/place-list.xml:1]: According to the schema, only permitted value of `ended` field is `true`.

File: mb-solr/src/test/java/org/musicbrainz/search/solrwriter/AbstractMBWriterRecordingTest.java [hunk 3]
```diff
@@ -9,7 +9,6 @@
 		return new ArrayList<>(Arrays.asList(new String[]{
 				"mbid", uuid,
 				"recording", "Roots and Beginnings",
-				"name", "Roots and Beginnings",
```
Comment (amCap1712) [inline mb-solr/src/test/java/org/musicbrainz/search/solrwriter/AbstractMBWriterRecordingTest.java:12]: I do not know what is the purpose of name here but removing it fixes the test.

File: mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml (hunk 1 above)
Comment (yvanzo) [reply mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml:1]: Not sure what you call “production” and “current” behaviors?

* [Search for this release](http://musicbrainz.org/ws/2/release/?query=reid:871f12ec-6259-4e93-b440-9108559c38b1&fmt=json) (currently in production) returns:

  ```json
  packaging: {
    "name": "None",
    "id": "119eba76-b343-3e02-a292-f0f00644bb9b"
  }
  ```

* [Lookup at this release](http://musicbrainz.org/ws/2/release/871f12ec-6259-4e93-b440-9108559c38b1?fmt=json) (currently in production) returns:

  ```json
  packaging: "None",
  packaging-id: "119eba76-b343-3e02-a292-f0f00644bb9b",
  ```

This is a known issue: SEARCH-579.

Comment (amCap1712) [reply mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml:1]: Yes, This is is what I was referring to. To be sure, we want that the Lookup behavior should be followed by the search as well.

File: mb-solr/src/test/java/org/musicbrainz/search/solrwriter/AbstractMBWriterRecordingTest.java (hunk 3 above)
Comment (yvanzo) [reply mb-solr/src/test/java/org/musicbrainz/search/solrwriter/AbstractMBWriterRecordingTest.java:12]: It was most likely a search field `name` redundant with the search field `recording` of [Recording](https://musicbrainz.org/doc/MusicBrainz_API/Search#Recording).

**Review by yvanzo — COMMENTED** [review]
Thanks for these changes!
The packaging can indeed be fixed later on, dropping it for now is fine by me.
Changes should preferably be made in separate commits with a reference to the related ticket if any.

File: mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml (hunk 1 above)
Comment (yvanzo) [reply mb-solr/src/test/resources/org/musicbrainz/search/solrwriter/release.xml:1]: Yes.

File: .travis.yml [hunk 4]
```diff
@@ -7,11 +7,11 @@ cache:
 before_install:
 - echo "MAVEN_OPTS='-Xmx1536m'" > ~/.mavenrc
 - pushd mmd-schema/brainz-mmd2-jaxb
-- mvn -DskipTests install
+- mvn install
 - popd
 - cd mb-solr
 after_success:
-- mvn package -DskipTests
+- mvn package
```
[the remark this answers is not in the document]
Comment (amCap1712) [reply .travis.yml]: These changes actually not needed and I made the second commit to revert those. I'll fix this.

**Comment by amCap1712** [comment]
I have split the commit into separate ones.

**Review by yvanzo — APPROVED** [review]
Thank you!

**Pull request merged**